Makefile

```diff
@@ -59,6 +59,9 @@ endif
 build-native:
 	bash ./scripts/build/build-native.sh
 
+build-deb:
+	cargo deb -p databend-binaries
```

Comment:
How about using https://nfpm.goreleaser.com/?

Reply:
Good idea, I'll update this PR during the night.